Heatington/ResultDataManager/ResultDataManager.cs

```diff
@@ -1,21 +1,56 @@
+using System.Buffers;
 using Heatington;
 using Heatington.Controllers;
 using Heatington.Models;
 using Heatington.Optimizer;
 
 namespace Heatington.ResultDataManager;
 
-public class ResultDataManager(CsvController _csvController)
+public class ResultDataManager
```

Comment:
so no dependency injection?

Reply:
When I am still updated we want to go singleton design pattern and full microservice with loose coupling. 

Therefore I assume that Francesco's solution with having only the constructor with 1 param and without contructor injection which is in this case filePath and the instantiation of the CsvController is preferable.